src/DrawModes/MoveMultiTool.ts

```diff
@@ -55,7 +55,7 @@ export function moveMultiMouseMove(event: MouseEvent) {
 
         redrawTree(treeContext.tree);
         if (currentNode instanceof CutNode) {
-            const color = validateChildren(currentNode, moveDifference)
+            const color = validateChildren(treeContext.tree, currentNode, moveDifference)
                 ? legalColor()
                 : illegalColor();
```

Comment:
Should never have a multiline ternary operator, give `validateChildren(treeContext.tree, currentNode, moveDifference)` its own line and assign it to a constant.

Reply:
Should I also try to do this for 
`currentParent =
                currentTree.getLowestNode(movePoint) instanceof AtomNode
                    ? currentTree.getLowestParent(movePoint)
                    : currentTree.getLowestNode(movePoint);`